<!DOCTYPE html>
<html lang="en-us">
  <head>
    <meta charset="utf-8">
<meta name="viewport" content="width=device-width, initial-scale=1.0">
<meta name="generator" content="Hugo 0.34" />


<title>&#34;While it may be true that Evolutionary Anthropologists consider themselves scientists and use the terms evolution and evolutionary...&#34; - Ed Hagen</title>
<meta property="og:title" content="&#34;While it may be true that Evolutionary Anthropologists consider themselves scientists and use the terms evolution and evolutionary...&#34; - Ed Hagen">



  







<link rel="stylesheet" href="/css/fonts.css" media="all">
<link rel="stylesheet" href="/css/main.css" media="all">



  </head>
  <body>
    <div class="wrapper">
      <header class="header">
        <nav class="nav">
  <a href="/" class="nav-logo">
    <img src="/images/logo.png"
         width="200"
         height="300"
         alt="Logo">
  </a>

  <ul class="nav-links">
    
    <li><a href="/about/">About</a></li>
    
    <li><a href="http://anthro.vancouver.wsu.edu/faculty/hagen/">Ed Hagen@WSU</a></li>
    
  </ul>
</nav>

      </header>


<main class="content" role="main">

  <article class="article">
    
    <span class="article-duration">28 min read</span>
    

    <h1 class="article-title">&#34;While it may be true that Evolutionary Anthropologists consider themselves scientists and use the terms evolution and evolutionary...&#34;</h1>

    
    <span class="article-date">2018/03/03</span>
    

    <div class="article-content">
      <style>
p.caption {
  font-size: 0.7em;
}
</style>
<p><a href="https://caseyroulette.weebly.com">Casey Roulette</a>, a former PhD student of mine who is now an assistant professor at San Diego State University, recently received an <a href="/pdf/RouletteEmail.pdf">email</a> from a member of the <a href="http://www.bio.sdsu.edu/faculty/faculty.html">Biology Department</a> who was irate that Casey’s evolutionary anthropology course, <em>Evolution of Human Nature</em>, was being considered to fulfill “Natural Sciences” GE reqs. She informed him that she had complained to her Chair, who in turn complained to the Dean of the College of Sciences, and that “Both will be preparing a letter to be sent to the GE committee indicating that the College of Sciences does not support this course as a GE course.” The <a href="/pdf/RouletteEmail.pdf">email concluded</a>:</p>
<blockquote>
<p>While it may be true that Evolutionary Anthropologists consider themselves scientists and use the terms evolution and evolutionary, the “Evolutionary Biology” represented in this course does not reflect or represent modern Evolutionary Biology as defined by Evolutionary Biologists.</p>
</blockquote>
<blockquote>
<p>In our meeting with the Instructor, who I know is an Assistant Professor, I thought it was not collegial to point out that his views on “evolutionary anthropology” are not considered accurate by those of us trained in Evolutionary Biology.</p>
</blockquote>
<blockquote>
<p>…</p>
</blockquote>
<blockquote>
<p>It is deeply concerning that students could leave this campus with such an erroneous understanding of such a fundamental scientific process.</p>
</blockquote>
<p>She attached <a href="/pdf/Evaluation.pdf">an evaluation from the SDSU Department of Biology</a>. The first part argued that, on programmatic grounds, the course did a good job fulfilling the Social Science reqs. I agree.</p>
<p>The second part, however, echoed the email, arguing on <em>scientific</em> grounds that the course did not qualify as a GE course in the Natural Sciences because it presented a view of biology that was incomplete, biased and incorrect.</p>
<p>That’s odd. If true, Casey’s course shouldn’t qualify for GE in either the Social or Natural Sciences. In fact, his course shouldn’t be taught at all.</p>
<p>SDSU biologists’ first concern was that Casey presents evolution as synonymous with adaptation:</p>
<blockquote>
<p>The “Evolutionary” approach presented in this class seems to present biological evolution as synonymous with adaptation via natural selection. Natural selection is but one of the five evolutionary forces that determine patterns of variation within and among species. For example, no information is provided on the very important role of Random Genetic Drift in shaping genetic variation.</p>
</blockquote>
<blockquote>
<p>The practice of interpreting biological patterns only through the lens of adaptation has been labeled the “Adaptionist program”. For four decades, biologists have recognized the folly of this approach. The phrase “The Fallacy of Intuitive Evolutionary Thinking” has also been used to describe the assumption that every biological feature has been optimized by selection. Evolutionary biologists accept the power of Darwinian natural selection but we do so understanding that natural selection is a complex process.</p>
</blockquote>
<p>Their second concern was that Casey does not discuss genetic drift, and its role in human evolution:</p>
<blockquote>
<p>There is no discussion of the Neutral and Nearly Neutral models of Evolution, which have been the dominant models of molecular evolution for 50 years.</p>
</blockquote>
<blockquote>
<p>It is also well established that Homo sapiens has an effective population size (Ne) of approximately 10,000. With such a small effective population size it is widely understood (among evolutionary biologists) that selection is inefficient in H. sapiens. Because of this evolutionary biologists would expect to see high levels of both fixed and segregating neutral and nearly neutral (slightly deleterious) alleles within and between human populations. Our most current understanding of genetic variation in does not fit in to the pan selectionist approach presented in this class.</p>
</blockquote>
<p>By “the pan selectionist approach” I suppose they are referring to the focus of Casey’s course, <a href="https://en.wikipedia.org/wiki/Human_behavioral_ecology">Human Behavioral Ecology (HBE)</a>.</p>
<p>The <a href="https://doi.org/10.1007/s10539-016-9557-8">debate</a> over adaptationism, however, is an ongoing <a href="https://plato.stanford.edu/entries/adaptationism/"><em>debate</em></a>, one that began more than a century ago. Each side has involved towering figures in evolutionary biology like Fisher, Wright, Williams, Hamilton, Maynard Smith, Gould, Lewontin, and Kimura.</p>
<p>The neutralist–selectionist debate — are patterns of genetic variation primarily explained by random genetic drift or natural selection? — is, again, a <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1513187/"><em>debate</em></a>.</p>
<p>Does Casey’s <a href="/pdf/ANTHProposal.pdf">syllabus</a> present both sides of these debates?</p>
<p>The assigned reading in Week 2 of the course is Stephen Jay Gould’s <a href="http://www.stephenjaygould.org/library/gould_art-of-story-telling.pdf"><em>Sociobiology: the art of storytelling</em></a>. It was Gould, of course, who with co-author Richard Lewontin, introduced the phrase “Adaptationist Programme” in their hugely influential article <a href="http://rspb.royalsocietypublishing.org/content/royprsb/205/1161/581.full.pdf"><em>The spandrels of San Marco and the Panglossian paradigm: a critique of the adaptationist programme</em></a>. One of Gould’s key points is the important role of genetic drift.</p>
<p>More importantly, one of two required books for the course is Laland and Brown’s <a href="https://books.google.com/books?id=2KcbFVBSxWYC&amp;lpg=PP2&amp;ots=0paM-7Ti3_&amp;dq=%22Sense%20and%20nonsense%3A%20Evolutionary%20perspectives%20on%20human%20behaviour%22&amp;lr&amp;pg=PA1#v=onepage&amp;q&amp;f=false"><em>Sense and nonsense: Evolutionary perspectives on human behaviour</em></a>. I would count Laland as a consistent critic of the adaptationist program (in favor of his alternative, <a href="https://doi.org/10.1017/S0140525X00002417">niche construction</a>, developed in collaboration with eminent population geneticist <a href="http://www-evo.stanford.edu/Feldman-cv.pdf">Marcus Feldman</a>) whose book is a good-faith effort to describe the controversies as they apply to human evolution. Laland and Brown do discuss neutral theory, drift and molecular evolution, especially their intriguing parallels with cultural evolution. Casey also assigned chapter 7 of <a href="https://global.oup.com/ushe/product/evolution-of-human-behavior-9780195333589">Evolution of Human Behavior</a>, by Agustin Fuentes, who I would also count as a <a href="https://doi.org/10.1017/S0140525X0235009X">critic of adaptationism</a>.</p>
<p>Although Casey’s course focuses on the evolution of the human behavioral phenotype and not molecular evolution, I see no evidence that the course, which also assigns Wrangham’s <a href="https://books.google.com/books/about/Demonic_Males.html?id=fP0c4b3jbMYC"><em>Demonic males</em></a>, is giving students a biased overview of the <a href="https://books.google.com/books/about/Defenders_of_the_Truth.html?id=uar4qh_ELuEC">debates</a> over adaptationism and neutralism vs. selectionism.</p>
<p>And it’s not quite true that neutral models “have been the dominant models of molecular evolution for 50 years.” Instead, the intense debate has perhaps resulted in a consensus. According to evolutionary geneticists <a href="http://dx.doi.org/10.1038/hdy.2016.55">Charlesworth and Charlesworth</a>, “From the late 1980s, the neutral theory came increasingly to be used as a null hypothesis, against which alternative hypotheses could be tested, including the models of the effects of selection on neutral or nearly neutral variability at linked sites….”, a point also made by <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1513187/">Masatoshi Nei</a> and Kimura himself.</p>
<p>At this point I should admit that I’m an unreconstructed <a href="http://www.nybooks.com/articles/1997/06/12/darwinian-fundamentalism/">ultra-Darwinian Fundamentalist</a> who, each night, reads his daughters passages from the Selfish Gene. <a href="http://anthro.vancouver.wsu.edu/media/PDF/Hagen_1999_The_functions_of_postpartum_depression.pdf">Depression is an adaptation!</a> <a href="https://dx.doi.org/10.3389%2Ffpsyt.2013.00142">Drug use is an adaptation!</a> This was my logo for the first iteration of this blog:</p>

<figure >
    
        <img src="/images/pangloss.png" />
    
    
</figure>

<p>Casey, what’s with the balanced overview? Did I teach you nothing?</p>
<p>In my view, critics of adaptationism have things backwards. For adaptationists, the question is not, do constraints and noise play an important role in organism structure? The question is, why don’t constraints and noise dominate organism structure? Constraints and noise permeate physical processes, yet organisms — intricate machines that surpass all human technology — somehow manage to make precise copies of themselves in hostile environments. It is exactly this <a href="https://archive.org/details/WhatIsLife-EdwardSchrodinger">problem</a>, as I discuss in a bit <a href="http://anthro.vancouver.wsu.edu/media/PDF/Buss2.2.pdf">more detail here</a>, that adaptationists are trying to solve.</p>
<!-- Calling on the [second law of thermodynamics](https://en.wikipedia.org/wiki/Second_law_of_thermodynamics), Schrödinger [pointed out](https://archive.org/details/WhatIsLife-EdwardSchrodinger) that: -->
<!-- >It is by avoiding the rapid decay into the inert state of ‘equilibrium’ that an organism appears so enigmatic; so much so, that from the earliest times of human thought some special non-physical or supernatural force (vis viva, entelechy) was claimed to be operative in the organism, and in some quarters is still claimed. -->
<!-- Drift and constraints only come into play if organisms survive and reproduce, and it is exactly this enormously difficult problem – how organisms survive and reproduce in a particular physical, biological, and social environment – that adaptationists are trying to solve. I discuss this a bit more [here](http://anthro.vancouver.wsu.edu/media/PDF/Buss2.2.pdf). -->
<p>The population genetics folks have it right: random genetic variation, and more generally, noise, by-products, constraints, and thermodynamically favored physics and chemistry, should always be the null hypotheses that an adaptationist hypothesis needs to beat.</p>
<p>Course approval at SDSU is a pretty trivial topic, but the evidence for noise vs. selection in the human genome is not. I therefore thought I would use this post as an opportunity to learn a bit more about it. It’s not meant to be a comprehensive review, just a taste of some major issues. I’m not a genetics guy — far from it — so if I make any mistakes, let me know in the comments.</p>
<div id="population-genetics-101" class="section level3">
<h3>Population genetics 101</h3>
<p>The SDSU biologists’ argument that, due to a small effective population size (<span class="math inline">\(N_e\)</span>), selection is “inefficient” in <em>H. sapiens</em> rests on a classic result from population genetics. Each generation is a sample of alleles from the previous generation. A particular allele might decrease the probability that individuals with the allele reproduce (negative selection), increase the probability (positive selection), or have no effect (neutral). This effect is quantified by the selection coefficient, <span class="math inline">\(s\)</span>, which is the difference in fitness, <span class="math inline">\(W\)</span>, between two alleles, the wild type, A, and a mutant type, B:</p>
<p><span class="math display">\[
s = W_B - W_A
\]</span></p>
<p>Even when <span class="math inline">\(s=0\)</span>, the frequencies of A and B will change at least a little bit from generation to generation due to random differences in the survival and reproduction of individuals with A vs. B. On average, however, the frequency of a neutral mutant allele in a new generation is, under some strong simplifying assumptions, simply the frequency of the allele in the original generation.</p>
<p>If the population is small, however, then the “sample size” is small. Just as we learn in statistics, the smaller the sample size, the larger the variance in the “sample estimate.” Thus, in small populations, the frequency of a neutral allele will bounce around from generation to generation more than it would in large populations, eventually going to either 0 (lost) or 1 (fixed). In particular, the frequency of a new mutation is small, so there’s a good chance that it won’t be “sampled,” and will thus be lost.</p>
<p>But what if the allele isn’t neutral? Naively, if an allele has a negative effect on fitness, <span class="math inline">\(s&lt;0\)</span>, then its frequency should go to 0, and if it has a positive effect on fitness, <span class="math inline">\(s&gt;0\)</span>, then its frequency should go to 1. Again, though, allele frequencies fluctuate randomly. If these fluctuations are often larger than the systematic effects of selection, then a deleterious allele can become fixed, and a beneficial allele can be lost. This is much more likely to happen when a population has a small effective population size (for review of the <span class="math inline">\(N_e\)</span> concept, see <a href="http://dx.doi.org/10.1038/nrg2526">Charlesworth 2009</a>). Generally, only when the magnitude of <span class="math inline">\(s\)</span> is greater than twice the reciprocal of effective population size will the fate of an allele will be dominated by its fitness effects:</p>
<p><span class="math display">\[
|s| &gt; \frac{1}{2N_e}
\]</span></p>
<p>Otherwise, its fate will be dominated by drift. This is what SDSU biologists were referring to: as <span class="math inline">\(N_e\)</span> decreases, the fate of alleles will increasingly be dominated by drift. The importance of selection also depends on <span class="math inline">\(s\)</span>, however, the fitness of a new allele relative to an existing allele.</p>
<p>The rate of evolution depends on the rate of production of new mutations, which is proportional to the mutation rate, <span class="math inline">\(\mu\)</span>, the census population size, <span class="math inline">\(N_c\)</span> (because bigger populations will produce more mutations), and the probability that a mutation spreads and becomes fixed (<a href="http://dx.doi.org/10.1016/j.tree.2013.09.009">Lanfear et al. 2014</a>):</p>
<p><span class="math display">\[
R = 2N_c \mu P
\]</span></p>
<p>The probability of fixation, in turn, depends on the selection coefficient of a mutation. Figure <a href="#fig:subrate">1</a>, from <a href="http://dx.doi.org/10.1016/j.tree.2013.09.009">Lanfear et al. 2014</a>, illustrates the relationship between <span class="math inline">\(N_e\)</span>, <span class="math inline">\(s\)</span>, and the rate of evolution — the substitution rate vs. the mutation rate — conveniently using parameters that approximate those for ancestral <em>Homo</em>. The top panels illustrate positive selection, and the bottom panels negative selection. The left-hand panels show that the substitution rate increases not only as <span class="math inline">\(N_e\)</span> increases, but also as <span class="math inline">\(s\)</span> increases. The right-hand panels depict the relationship between the ratio of substitution rate to the mutation rate (on the y-axis) and <span class="math inline">\(N_es\)</span>. For <span class="math inline">\(N_es &lt; 1\)</span> (to the left of the dotted line), the substitution rate is approximately the mutation rate (drift). For <span class="math inline">\(N_es &gt; 1\)</span> (to the right of the dotted line), the substitution rate either begins to skyrocket beyond the mutation rate (positive selection) or drop to 0 (negative selection).</p>
<div class="figure"><span id="fig:subrate"></span>
<img src="/images/Lanfear2014.png" alt="The relationship between substitution rate (in substitutions per site per year) and effective population size ($N_e$) under genetic drift and natural selection (the $N_eRR$) [8]. These relationships were calculated assuming a mutation rate of $1 \times 10^{-9}$ mutations per site per year, approximately that found in humans. (A,B) show the substitution rate of mutations for a range of positive (A) and negative (B) selection coefficients (denoted ‘s’). (C,D) show the same data, but in this case the y-axis shows the substitution rate relative to the mutation rate, and the x-axis shows the product of $N_e$ and the selection coefficient for positive (C) and negative (D) mutations respectively. A dashed line highlights where $N_es = 1$, below which mutations are often considered ‘effectively neutral’. Note that genetic drift predicts a flat $N_eRR$ for neutral mutations, where $s = 0.00$ in (A,B). In (C,D), this is reflected by the substitution rate equaling the mutation rate, giving a value of 1 on the y-axis, when $N_es = 0$. Figure and Caption from [Lanfear et al. 2014](http://dx.doi.org/10.1016/j.tree.2013.09.009)."  />
<p class="caption">
Figure 1: The relationship between substitution rate (in substitutions per site per year) and effective population size (<span class="math inline">\(N_e\)</span>) under genetic drift and natural selection (the <span class="math inline">\(N_eRR\)</span>) [8]. These relationships were calculated assuming a mutation rate of <span class="math inline">\(1 \times 10^{-9}\)</span> mutations per site per year, approximately that found in humans. (A,B) show the substitution rate of mutations for a range of positive (A) and negative (B) selection coefficients (denoted ‘s’). (C,D) show the same data, but in this case the y-axis shows the substitution rate relative to the mutation rate, and the x-axis shows the product of <span class="math inline">\(N_e\)</span> and the selection coefficient for positive (C) and negative (D) mutations respectively. A dashed line highlights where <span class="math inline">\(N_es = 1\)</span>, below which mutations are often considered ‘effectively neutral’. Note that genetic drift predicts a flat <span class="math inline">\(N_eRR\)</span> for neutral mutations, where <span class="math inline">\(s = 0.00\)</span> in (A,B). In (C,D), this is reflected by the substitution rate equaling the mutation rate, giving a value of 1 on the y-axis, when <span class="math inline">\(N_es = 0\)</span>. Figure and Caption from <a href="http://dx.doi.org/10.1016/j.tree.2013.09.009">Lanfear et al. 2014</a>.
</p>
</div>
<p>So, for humans, we need two pieces of evidence, (1) <span class="math inline">\(N_e\)</span>, and (2) the distribution of fitness effects (DFE) for new alleles (there is debate about the human mutation rate; <a href="(http://dx.doi.org/10.1038/nrg3295)">Scally and Durbin 2012</a>; <a href="http://dx.doi.org/10.1016/j.gde.2016.07.008">Scally 2016</a>).</p>
</div>
<div id="empirical-estimates-of-effective-population-size-n_e" class="section level3">
<h3>Empirical estimates of effective population size (<span class="math inline">\(N_e\)</span>)</h3>
<p>Like all other lineages, the human lineage extends back to the origin of life, <span class="math inline">\(&gt;3\)</span> billion years ago. It is therefore informative to consider the <span class="math inline">\(N_e\)</span> of our lineage over different periods of our evolution. Here is a relatively recent estimate of divergence times and <span class="math inline">\(N_e\)</span>’s for the great apes, including humans, from <a href="http://dx.doi.org/10.1038/nature12228">Prado-Martinez et al. (2013)</a>:</p>
<div class="figure"><span id="fig:apedivergence"></span>
<img src="/images/nature12228-f2.2.jpg" alt="Population splits and effective population sizes ($N_e$) during great ape evolution. Split times (dark brown) and divergence times (light brown) are plotted as a function of divergence (d) on the bottom and time on top. Time is estimated using a single mutation rate (μ) of $1 \times 10^{-9}$ $mut$ $bp^{−1}$ $year^{−1}$. The ancestral and current effective population sizes are also estimated using this mutation rate. The results from several methods used to estimate $N_e$ (COALHMM, ILS COALHMM, PSMC and ABC) are coloured in orange, purple, blue and green, respectively. The chimpanzee split times are estimated using the ABC method. The x axis is rescaled for divergences larger than $2 \times 10^{-3}$ to provide more resolution in recent splits. All the values used in this figure can be found in Supplementary Table 5. The terminal $N_e$ correspond to the effective population size after the last split event. Figure and caption from [Prado-Martinez et al. (2013)](http://dx.doi.org/10.1038/nature12228)."  />
<p class="caption">
Figure 2: Population splits and effective population sizes (<span class="math inline">\(N_e\)</span>) during great ape evolution. Split times (dark brown) and divergence times (light brown) are plotted as a function of divergence (d) on the bottom and time on top. Time is estimated using a single mutation rate (μ) of <span class="math inline">\(1 \times 10^{-9}\)</span> <span class="math inline">\(mut\)</span> <span class="math inline">\(bp^{−1}\)</span> <span class="math inline">\(year^{−1}\)</span>. The ancestral and current effective population sizes are also estimated using this mutation rate. The results from several methods used to estimate <span class="math inline">\(N_e\)</span> (COALHMM, ILS COALHMM, PSMC and ABC) are coloured in orange, purple, blue and green, respectively. The chimpanzee split times are estimated using the ABC method. The x axis is rescaled for divergences larger than <span class="math inline">\(2 \times 10^{-3}\)</span> to provide more resolution in recent splits. All the values used in this figure can be found in Supplementary Table 5. The terminal <span class="math inline">\(N_e\)</span> correspond to the effective population size after the last split event. Figure and caption from <a href="http://dx.doi.org/10.1038/nature12228">Prado-Martinez et al. (2013)</a>.
</p>
</div>
<p>And here is a recent estimate of <span class="math inline">\(N_e\)</span> for modern <em>H. sapiens</em> from about 200,000 years ago to the present, from <a href="https://dx.doi.org/10.1038%2Fng.3015">Schiffels and Durbin (2014)</a>.</p>
<div class="figure"><span id="fig:ne"></span>
<img src="/images/NeAB.png" alt="Estimates of $N_e$ for humans. The blue colors are African populations, and the other colors are non-African populations. The top panel used different data that included Native American haplotypes, with better resolution for older population changes than the bottom panel, but less resolution for more recent changes. Figure from [Schiffels and Durbin (2014)](https://dx.doi.org/10.1038%2Fng.3015)."  />
<p class="caption">
Figure 3: Estimates of <span class="math inline">\(N_e\)</span> for humans. The blue colors are African populations, and the other colors are non-African populations. The top panel used different data that included Native American haplotypes, with better resolution for older population changes than the bottom panel, but less resolution for more recent changes. Figure from <a href="https://dx.doi.org/10.1038%2Fng.3015">Schiffels and Durbin (2014)</a>.
</p>
</div>
<p>Thus, based on estimated <span class="math inline">\(N_e\)</span>, and assuming a fixed DFE, the rate of evolution should have been relatively high when the great apes diverged from other apes about 20 million years ago, slowed when hominins diverged from chimpanzees at the end of the Miocene, slowed again, perhaps with the appearance of <em>Homo</em> around the beginning of the Pleistocene, and then very recently accelerated in the late Pleistocene and Holocene.</p>
</div>
<div id="the-distribution-of-fitness-effects-dfe" class="section level3">
<h3>The distribution of fitness effects (DFE)</h3>
<p>The DFE of new mutations, however, is not necessarily fixed. Generally, deleterious mutations are expected to greatly outnumber beneficial ones because there are more ways to break things than improve them, especially if a population is well-adapted to its environmental niche.</p>
<p>If a population is moving into a new environmental niche, however (point A in Figure <a href="#fig:DFE">4</a>), the fitness of existing alleles might drop, pushing the distribution of fitness effects of new alleles to higher values (distributions on the left). As the population adapts and reaches a fitness maximum (point B), new mutations will almost always be detrimental (distributions on the right).</p>
<div class="figure"><span id="fig:DFE"></span>
<img src="/images/Lanfear2014.2.png" alt="Fitness landscapes and the distribution of fitness effects. Hypothetical distributions of fitness effects for populations of different sizes at different positions on a simple fitness landscape. A population far from the optimum (A) has a certain proportion of mutations that confer increases in fitness. However, a population at the hypothetical optimum of the landscape (B) cannot increase it fitness, so all mutations are deleterious. In both cases, the proportion of mutations that fall into different categories (Box 2, main text) changes depending on the effective population size. Note that, for simplicity, we have drawn a fitness landscape that varies along a single dimension, but the distributions we have drawn are more similar to those that would come from higher-dimensional fitness landscapes. Furthermore, it is unlikely that any natural population sits at the precise optimum of any fitness landscape. The selection coefficients are shown on natural scales, not log-transformed scales. Figure and caption from [Lanfear et al. 2014](http://dx.doi.org/10.1016/j.tree.2013.09.009)."  />
<p class="caption">
Figure 4: Fitness landscapes and the distribution of fitness effects. Hypothetical distributions of fitness effects for populations of different sizes at different positions on a simple fitness landscape. A population far from the optimum (A) has a certain proportion of mutations that confer increases in fitness. However, a population at the hypothetical optimum of the landscape (B) cannot increase it fitness, so all mutations are deleterious. In both cases, the proportion of mutations that fall into different categories (Box 2, main text) changes depending on the effective population size. Note that, for simplicity, we have drawn a fitness landscape that varies along a single dimension, but the distributions we have drawn are more similar to those that would come from higher-dimensional fitness landscapes. Furthermore, it is unlikely that any natural population sits at the precise optimum of any fitness landscape. The selection coefficients are shown on natural scales, not log-transformed scales. Figure and caption from <a href="http://dx.doi.org/10.1016/j.tree.2013.09.009">Lanfear et al. 2014</a>.
</p>
</div>
<p>There are good reasons to believe that our lineage has undergone multiple niche changes, e.g., when hominins diverged from the chimpanzee lineage toward the end of the Miocene, when <em>Homo</em> diverged from other hominins around the beginning of the Pleistocene, when modern humans left Africa c. 50-100,000 years ago, and when transitioning to agriculture at the end of the Pleistocene and beginning of the Holocene c. 10,000 years ago. During these changes, the DFE could have been shifted to higher values, resulting in a higher rate of evolution.</p>
<p>Several other mechanisms have been proposed that might either change or stabilize the DFE in different species with different <span class="math inline">\(N_e\)</span> and different levels of organism complexity (Figure <a href="#fig:DFEmodels">5</a>).</p>
<div class="figure"><span id="fig:DFEmodels"></span>
<img src="/images/DFE_models.png" alt="Overview of the main predictions of five theoretical models regarding DFE differences between two species. Here, E[s] is the average selection coefficient of a new mutation, and $N_e$ is the effective population size. Figure and caption from [Huber et al. 2017](https://doi.org/10.1073/pnas.1619508114)."  />
<p class="caption">
Figure 5: Overview of the main predictions of five theoretical models regarding DFE differences between two species. Here, E[s] is the average selection coefficient of a new mutation, and <span class="math inline">\(N_e\)</span> is the effective population size. Figure and caption from <a href="https://doi.org/10.1073/pnas.1619508114">Huber et al. 2017</a>.
</p>
</div>
<p><a href="https://doi.org/10.1073/pnas.1619508114">Huber et al. 2017</a>, for example, found that polymorphism data from humans, <em>Drosophila</em>, mice, and yeast best supported Fisher’s Geometrical Model, which represents phenotypes as points in a multidimensional phenotype space, whose dimensionality is termed “complexity.” Fitness is a decreasing function of the distance from the optimal phenotype. Because mutations in complex organisms are more likely to disrupt functionality, the average selection coefficient should be more negative in complex vs. simple organisms, a prediction supported by their data (Figure <a href="#fig:FGM">6</a>, panel A):</p>
<div class="figure"><span id="fig:FGM"></span>
<img src="/images/FGM.png" alt="Empirical support for FGM. (A) Both under the gamma DFE and the Lourenço et al. DFE, estimated average deleteriousness of mutations increases as a function of organismal complexity. (B) The shape parameter of the gamma DFE depends on the breadth of gene expression. Tissue-specific genes have a smaller shape parameter (α) than broadly expressed genes, supporting FGM. This pattern is consistent across overall expression levels. (C and D) By fitting the DFE of Lourenço et al., we can model slightly beneficial mutations in the DFE (green) that are thought to compensate for fixed deleterious mutations in species with small population size. We find support for a larger proportion of slightly beneficial mutations in the DFE of (C) humans than in (D) Drosophila. Figure and caption from [Huber et al. 2017](https://doi.org/10.1073/pnas.1619508114)."  />
<p class="caption">
Figure 6: Empirical support for FGM. (A) Both under the gamma DFE and the Lourenço et al. DFE, estimated average deleteriousness of mutations increases as a function of organismal complexity. (B) The shape parameter of the gamma DFE depends on the breadth of gene expression. Tissue-specific genes have a smaller shape parameter (α) than broadly expressed genes, supporting FGM. This pattern is consistent across overall expression levels. (C and D) By fitting the DFE of Lourenço et al., we can model slightly beneficial mutations in the DFE (green) that are thought to compensate for fixed deleterious mutations in species with small population size. We find support for a larger proportion of slightly beneficial mutations in the DFE of (C) humans than in (D) Drosophila. Figure and caption from <a href="https://doi.org/10.1073/pnas.1619508114">Huber et al. 2017</a>.
</p>
</div>
<p><a href="https://doi.org/10.1371/journal.pgen.1004697">Racimo and Schraiber (2014)</a> criticize empirical studies of DFE that rely on fitting the data to a single probability distribution, however, such as normal or gamma. They found, instead, that in humans the DFE (for deleterious mutations only) had a bimodal distribution, with a large peak centered on <span class="math inline">\(s \sim 0\)</span> (neutrality), and smaller peak between <span class="math inline">\(-10^{-5}\)</span> and <span class="math inline">\(-10^{-4}\)</span>:</p>
<div class="figure"><span id="fig:racimo"></span>
<img src="/images/journal.pgen.1004697.g002.png" alt="Distribution of fitness effects among YRI polymorphisms in the Complete Genomics dataset, partitioned by the genomic consequence of the mutated site. The right panels show a zoomed-in version of the distributions in the left panels, after removing neutral polymorphisms and log-scaling the x-axis. A) DFE obtained from the genome-wide mapping. B) Zoomed-in version of panel A. C) DFE obtained from the exome-wide mapping. D) Zoomed-in version of panel C. E) DFEs for exonic sites (nonsynonymous, synonymous, splice sites) obtained from the exome-wide mapping and DFEs for non-exonic sites (intergenic, UTR, regulatory) obtained from the genome-wide mapping. F) Zoomed-in version of panel E. Consequences were determined using the Ensembl Variant Effect Predictor (v.2.5). Codon and degeneracy information was obtained from snpEff. If more than one consequence existed for a given SNP, that SNP was assigned to the most severe of the predicted categories, following the VEP's hierarchy of consequences. NonSyn = nonsynonymous. Syn = synonymous. Syn to unpref. codon = synonymous change from a preferred to an unpreferred codon. Syn to pref. codon = synonymous change from an unpreferred to a preferred codon. Syn no pref. = synonymous change from an unpreferred codon to a codon that is also unpreferred. Splice = splice site. Figure and caption from [Racimo and Schraiber (2014)](https://doi.org/10.1371/journal.pgen.1004697). https://doi.org/10.1371/journal.pgen.1004697.g002"  />
<p class="caption">
Figure 7: Distribution of fitness effects among YRI polymorphisms in the Complete Genomics dataset, partitioned by the genomic consequence of the mutated site. The right panels show a zoomed-in version of the distributions in the left panels, after removing neutral polymorphisms and log-scaling the x-axis. A) DFE obtained from the genome-wide mapping. B) Zoomed-in version of panel A. C) DFE obtained from the exome-wide mapping. D) Zoomed-in version of panel C. E) DFEs for exonic sites (nonsynonymous, synonymous, splice sites) obtained from the exome-wide mapping and DFEs for non-exonic sites (intergenic, UTR, regulatory) obtained from the genome-wide mapping. F) Zoomed-in version of panel E. Consequences were determined using the Ensembl Variant Effect Predictor (v.2.5). Codon and degeneracy information was obtained from snpEff. If more than one consequence existed for a given SNP, that SNP was assigned to the most severe of the predicted categories, following the VEP’s hierarchy of consequences. NonSyn = nonsynonymous. Syn = synonymous. Syn to unpref. codon = synonymous change from a preferred to an unpreferred codon. Syn to pref. codon = synonymous change from an unpreferred to a preferred codon. Syn no pref. = synonymous change from an unpreferred codon to a codon that is also unpreferred. Splice = splice site. Figure and caption from <a href="https://doi.org/10.1371/journal.pgen.1004697">Racimo and Schraiber (2014)</a>. <a href="https://doi.org/10.1371/journal.pgen.1004697.g002" class="uri">https://doi.org/10.1371/journal.pgen.1004697.g002</a>
</p>
</div>
<p><a href="https://doi.org/10.1371/journal.pgen.1004697">Racimo and Schraiber (2014)</a> speculate that the absence of mutations with <span class="math inline">\(s &lt; -10^{-4}\)</span> might indicate a cutoff between weakly deleterious mutations that segregate in human populations and highly deleterious mutations that are quickly eliminated by negative selection.</p>
<p>In summary, the rate of evolution depends on both the DFE and <span class="math inline">\(N_e\)</span>, and involves both negative (purifying) and positive selection; the DFE likely depends on changes to the environment, organism complexity, and perhaps other other factors like robustness and back mutations; and the human DFE is an active area of research.</p>
</div>
<div id="standing-variation-and-soft-sweeps" class="section level3">
<h3>Standing variation and soft sweeps</h3>
<p>An important further consideration is that the stochastic effects of drift on the fate of beneficial alleles apply mainly to new mutations, because they are initially at very low frequency. Species harbor a large number of neutral, or nearly neutral, alleles, however, termed standing variation, which, because they are already at high frequency, are much less likely to be lost via drift. Moreover, these alleles already exist, so there is no waiting time for new beneficial mutations to appear. When a population moves into a new niche, these formerly neutral alleles can become either deleterious or beneficial and will respond quickly to selection, resulting in a soft sweep <a href="https://doi.org/10.1534/genetics.104.036947">(Hermisson and Pennings 2005)</a>.</p>
<p>Figure <a href="#fig:softsweep">8</a>, from <a href="https://doi.org/10.1016/j.tree.2007.09.008">Barrett and Schluter 2008</a>, illustrates that, for smaller values of <span class="math inline">\(\alpha_b = 2N_es_b\)</span> (where the <span class="math inline">\(b\)</span> subscript indicates “beneficial”), beneficial standing variants (solid line) have a very high fixation probability relative to new mutations (dashed line):</p>
<div class="figure"><span id="fig:softsweep"></span>
<img src="/images/Barrett2008.jpg" alt="The probability of fixation of a single new mutation (dashed curve) compared with that of a polymorphic allele that arose in a single mutational event (solid curve). $\alpha_b = 2N_es_b$, where $N_e$ is the effective population size and $s_b$ is the homozygous fitness advantage. The form of the curve for standing variation in this example assumes that $N = N_e = 25,000$, the dominance coefficient (h) = 0.5 and that beneficial alleles were previously neutral. $\alpha_b$ is plotted on a logarithmic scale. Figure and caption from [Barrett and Schluter 2008](https://doi.org/10.1016/j.tree.2007.09.008)."  />
<p class="caption">
Figure 8: The probability of fixation of a single new mutation (dashed curve) compared with that of a polymorphic allele that arose in a single mutational event (solid curve). <span class="math inline">\(\alpha_b = 2N_es_b\)</span>, where <span class="math inline">\(N_e\)</span> is the effective population size and <span class="math inline">\(s_b\)</span> is the homozygous fitness advantage. The form of the curve for standing variation in this example assumes that <span class="math inline">\(N = N_e = 25,000\)</span>, the dominance coefficient (h) = 0.5 and that beneficial alleles were previously neutral. <span class="math inline">\(\alpha_b\)</span> is plotted on a logarithmic scale. Figure and caption from <a href="https://doi.org/10.1016/j.tree.2007.09.008">Barrett and Schluter 2008</a>.
</p>
</div>
<p>In a study that used a novel machine learning technique to identify hard sweeps (new mutations that go to fixation under positive selection) and soft sweeps across the human genome, <a href="https://doi.org/10.1093/molbev/msx154">Schrider and Kern (2017)</a> found that the vast majority, over 90%, were soft sweeps. In addition, patterns of variation in perhaps half the genome has been affected by a nearby sweep:</p>
<div class="figure"><span id="fig:hardsoft"></span>
<img src="/images/msx154f2.png" alt="The number of windows assigned to each class by S/HIC in each population."  />
<p class="caption">
Figure 9: The number of windows assigned to each class by S/HIC in each population.
</p>
</div>
<p>This is a new technique based on simulated training data. Schrider and Kern acknowledge that their results</p>
<blockquote>
<p>may be surprising given the apparently small effective population size and low nucleotide diversity levels in humans. However, if the mutational target for the trait to be selected on is fairly large, then the probability of a population harboring a mutation affecting that trait may be appreciable.</p>
</blockquote>
<p><em>Variance</em> in <span class="math inline">\(N_e\)</span> can also be important in determining the relative importance of hard vs. soft sweeps (<a href="https://doi.org/10.1016/j.tree.2013.08.003">Messer and Petrov 2013</a>). <span class="math inline">\(N_e\)</span> estimated from sequence data can be dominated by short phases during which <span class="math inline">\(N_e\)</span> was small, even though <span class="math inline">\(N_e\)</span> was large for long periods during which adaptation by positive selection was much more likely.</p>
</div>
<div id="the-effects-of-population-structure-on-the-substitution-rate" class="section level3">
<h3>The effects of population structure on the substitution rate</h3>
<p>Another consideration is that the population genetics models of the effects of <span class="math inline">\(N_e\)</span> and <span class="math inline">\(s\)</span> on substitution rates make simplifying assumptions, such as random mating (i.e., a lack of population structure), that might easily be violated in real populations. Recent theoretical work has found that population structure can either suppress or amplify the effects of selection (e.g., <a href="http://dx.doi.org/10.1098/rspb.2013.0211">Frean et al. 2013</a>).</p>
</div>
<div id="phenotypic-evidence-for-human-evolution" class="section level3">
<h3>Phenotypic evidence for human evolution</h3>
<p>The final piece of background information to keep in mind is the physiological, morphological, and behavioral evidence for human evolution that has informed the studies of evolutionary biologists from Charles Darwin in the 19th century to <a href="http://ecologyandevolution.cornell.edu/faculty">entire departments</a> of evolutionary biologists in the 21st.</p>
<p>Perhaps the two most dramatic phenotypic examples of human-specific adaptations are (1) bipedalism, and (2) the substantial increase in brain size in humans relative to hominin ancestors and extant apes, most of which occurred since the first appearance of <em>Homo</em> over 2 million years ago (Figure <a href="#fig:brainsize">10</a>):</p>
<div class="figure"><span id="fig:brainsize"></span>
<img src="/images/brainsize.png" alt="The evolution of primate cranial capacity. Each dot is a fossil specimen. The x-axis is on a log scale. From [Schoenemann (2013)](http://dx.doi.org/10.1002/9781118332344.ch8)"  />
<p class="caption">
Figure 10: The evolution of primate cranial capacity. Each dot is a fossil specimen. The x-axis is on a log scale. From <a href="http://dx.doi.org/10.1002/9781118332344.ch8">Schoenemann (2013)</a>
</p>
</div>
<p>Two recent studies of variation in cranial dimensions in apes and humans (<a href="http://dx.doi.org/10.1098/rspb.2015.1519">Weaver and Stringer 2015</a>; <a href="http://dx.doi.org/10.1111/evo.13361">Schroeder and von Cramon-Taubadel 2017</a>) found that, whereas great ape cranial evolution was largely characterized by strong stabilizing selection, the divergence of <em>Homo</em> from its last common ancestor with chimpanzees was explained by strong directional selection.</p>
<p>The ability to learn language is a widely accepted cognitive difference between humans and chimpanzees that is probably rooted in human encephalization, as might be the cognitive abilities underlying cumulative culture, which is arguably a uniquely human trait (for a review of the evidence for cumulative culture in humans and other animals, see <a href="http://dx.doi.org/10.1111/brv.12053">Dean et al. 2013</a>).</p>
<p>Detailed comparisons of other important aspects of human and great ape phenotypes are available in the <a href="https://carta.anthropogeny.org/moca/domains">Matrix of Comparative Anthropogeny (MOCA)</a> of the Center for Academic Training and Research in Anthropogeny (CARTA).</p>
</div>
<div id="empirical-evidence-that-links-genetic-evolution-with-phenotype-evolution" class="section level2">
<h2>Empirical evidence that links genetic evolution with phenotype evolution</h2>
<p>We are only at the beginning of very long quest to link phenotypes, including the <a href="https://carta.anthropogeny.org/moca/domains/behavior">behavioral</a> and <a href="https://carta.anthropogeny.org/moca/domains/culture">cultural</a> phenotypes that are the topic of Casey’s course, to the genome. Nevertheless, we know a lot more today than we knew 10 or even 5 years ago.</p>
<div id="functional-vs.junk-dna" class="section level3">
<h3>Functional vs. junk DNA</h3>
<p>Genomes can be divided into two parts: functional DNA, which plays a profound role in the phenotype, and “junk” DNA, which plays no role in the phenotype. Functional DNA comprises protein coding regions and non-coding regulatory regions. We now have a pretty good understanding of which DNA sequences code for protein. It is much more difficult, however, to distinguish regulatory DNA from “junk” DNA, and hence to determine the fraction of the genome that is functional.</p>
<p>Most attempts to distinguish functional from junk DNA involve identifying sequences that are conserved across species, and are therefore presumed to be under purifying selection (constrained sequences) and thus functional, vs. those that are not conserved across species and so presumably have not been under purifying selection and are therefore likely non-functional “junk” (c.f., <a href="http://blogs.nature.com/news/2012/09/fighting-about-encode-and-junk.html">ENCODE</a>).</p>
<p>A recent estimate (<a href="https://doi.org/10.1371/journal.pgen.1004525">Rands et al. 2014</a>) is that 8.2% (7.1–9.2%) of the human genome is functional (constrained). Protein coding sequences comprise about 1% of the genome, and are highly conserved across the mammals (Figure <a href="#fig:cvalue">11</a>, red). Non-coding regulatory sequences have much higher rates of turnover (turnover refers to the loss or gain of purifying selection at a particular locus of the genome caused by changes in the physical or genetic environment, or mutations at the locus itself, that switch it from being functional to being non-functional or vice versa). See Figure <a href="#fig:cvalue">11</a>:</p>
<div class="figure"><span id="fig:cvalue"></span>
<img src="/images/journal.pgen.1004525.g004.png" alt="Schematic summary of the fraction of constrained sequence that has been retained (saturated colours) or turned over (pastel colours) in the human lineage over time (X-axis, divergence time) and how it has been distributed across various categories of functional element. In addition to showing the reduced quantity of preserved constrained sequence with increasing divergence, we infer the reciprocal quantity of sequence that is assumed to have been gained over human lineage evolution. Figure and caption from [Rands et al. 2014](https://doi.org/10.1371/journal.pgen.1004525)."  />
<p class="caption">
Figure 11: Schematic summary of the fraction of constrained sequence that has been retained (saturated colours) or turned over (pastel colours) in the human lineage over time (X-axis, divergence time) and how it has been distributed across various categories of functional element. In addition to showing the reduced quantity of preserved constrained sequence with increasing divergence, we infer the reciprocal quantity of sequence that is assumed to have been gained over human lineage evolution. Figure and caption from <a href="https://doi.org/10.1371/journal.pgen.1004525">Rands et al. 2014</a>.
</p>
</div>
<p>Because <span class="math inline">\(&gt;90\%\)</span> of the human genome appears to be non-functional, sequence variation in this portion should closely follow the neutral model. Our focus, then, is on the evolution of the <span class="math inline">\(\sim 8\%\)</span> of the genome that is functional.</p>
</div>
<div id="conserved-traits" class="section level3">
<h3>Conserved traits</h3>
<p>Perhaps the most important point is that evolutionary anthropologists are keenly interested in the adaptations we <em>share</em> with primates, mammals, vertebrates, and so on. Examples include <a href="http://dx.doi.org/10.1002/evan.20289">lacation</a>, the <a href="http://dx.doi.org/10.1002/ajhb.21092">immune system</a>, <a href="http://dx.doi.org/10.1002/evan.20195">vision</a> and bitter taste and other plant toxin defense mechanisms that are central to <a href="(https://doi.org/10.3389/fpsyt.2013.00142)">Casey’s</a> <a href="https://doi.org/10.1016/j.jep.2018.01.022">research</a>. The major features of these adaptations evolved long before the appearance of <em>Homo</em>, with its small <span class="math inline">\(N_e\)</span>, but would need to have been maintained by purifying selection during the evolution of <em>Homo</em>.</p>
<p>Much of the genetic basis of adaptations we share with other mammals and primates almost certainly lies in the <span class="math inline">\(\sim 2\%\)</span> of constrained sequence we share with all other mammals and the <span class="math inline">\(\sim 6-7\%\)</span> we share with other primates (out of 8.2% total; Figure <a href="#fig:cvalue">11</a>).</p>
<p>Moreover, many complex adaptations have evolved features that enable them to adjust ontogenetically to local environmental conditions, an ability that I and many others have <a href="http://anthro.vancouver.wsu.edu/media/PDF/Hagen_and_Hammerstein_2005_Evo_Devo.pdf">framed in strategic terms</a>. Examples include the immune system and <a href="https://www.ncbi.nlm.nih.gov/pubmed/18288955">induction of xenobiotic metabolizing enzymes</a>. Humans would not be doing too horribly in a wide range of environments even if there were no human-specific adaptations.</p>
</div>
<div id="human-specific-selection-in-regulatory-sequences" class="section level3">
<h3>Human-specific selection in regulatory sequences</h3>
<p>Given that protein-coding sequences are highly conserved across the mammals, and that most functional DNA comprises regulatory sequences, most human-specific adaptations should be grounded in changes to regulatory sequences.</p>
<p>Human accelerated regions (HARs) are short, evolutionarily conserved DNA sequences that have acquired significantly more DNA substitutions than expected in the human lineage since divergence from chimpanzees. HARs are often, but not always, the product of positive natural selection; other mechanisms include relaxation of constraint, which allows a region to acquire more mutations than it would under purifying selection, and GC-biased gene conversion (<a href="http://doi.org/10.1186/s12915-017-0428-9">Franchini and Pollard 2017</a>).</p>
<p><a href="http://doi.org/10.1186/s12915-017-0428-9">Franchini and Pollard 2017</a> summarized multiple studies that attempted to discover HARs, which differed in the number of species considered to determine “conserved” (which ranged from a few primate species to multiple primate, mammalian and vertebrate species), sequence filtering criteria (e.g., including or excluding coding sequences), and statistical tests for acceleration. Although each study identified a large number of HARs, there was only limited overlap in the identified regions. See Figure <a href="#fig:HAR">12</a>:</p>
<div class="figure"><span id="fig:HAR"></span>
<img src="/images/HAR.png" alt="Identification of human accelerated elements. Top: the four different approaches used to identify human accelerated regions. Some key differences include (i) the conserved elements used as candidates to identify HARs (which depend on multiple sequence alignments, methods to detect conservation, and whether human was masked in the alignments), (ii) bioinformatics filters that aim to restrict to non-coding elements and/or remove assembly or alignment artifacts, and (iii) tests used to detect acceleration. Bottom: overlap of the different datasets of human accelerated regions. Abbreviations: ANC accelerated conserved non-coding sequences [20]; HACNS human accelerated conserved non-coding sequences [23]; HTBE human terminal branch elements [21]. HARs include the original HARs [19] and the second generation HARs or 2xHARs [100]. Figure and caption from [Franchini and Pollard 2017](http://doi.org/10.1186/s12915-017-0428-9)."  />
<p class="caption">
Figure 12: Identification of human accelerated elements. Top: the four different approaches used to identify human accelerated regions. Some key differences include (i) the conserved elements used as candidates to identify HARs (which depend on multiple sequence alignments, methods to detect conservation, and whether human was masked in the alignments), (ii) bioinformatics filters that aim to restrict to non-coding elements and/or remove assembly or alignment artifacts, and (iii) tests used to detect acceleration. Bottom: overlap of the different datasets of human accelerated regions. Abbreviations: ANC accelerated conserved non-coding sequences [20]; HACNS human accelerated conserved non-coding sequences [23]; HTBE human terminal branch elements [21]. HARs include the original HARs [19] and the second generation HARs or 2xHARs [100]. Figure and caption from <a href="http://doi.org/10.1186/s12915-017-0428-9">Franchini and Pollard 2017</a>.
</p>
</div>
<p>Most studies of HARs used only sequence data. Some recent studies, though, also incorporated expression data. <a href="http://doi.org/10.1101/gr.192591.115">Gittelman et al. 2015</a>, for example, used maps of <a href="https://en.wikipedia.org/wiki/DNase_I_hypersensitive_site">DNase I hypersensitive sites (DHSs)</a> from <a href="https://www.encodeproject.org">ENCODE</a> and the <a href="http://www.roadmapepigenomics.org">Roadmap Epigenomics Projects</a>. DHSs are regions of <a href="https://en.wikipedia.org/wiki/Chromatin">chromatin</a> that serve as markers of regulatory DNA. Of 2,093,197 DHS loci, 113,577 exhibited significant constraint across the primates. DHSs active in fetal cell types, especially fetal brain cells, showed the highest levels of conservation in non-human primates. Of the conserved loci, 524 were accelerated in human evolution (haDHSs), evolving at approximately four times the neutral rate in the human lineage, mostly but not exclusively under positive selection, while other primate lineages evolved at less than half of the neutral rate. Gittelman et al. found that haDHSs tend to target developmentally and neuronally important genes relative to conserved DHSs, which themselves are already highly enriched for these categories.</p>
<p>In another review of several previous studies of HARs, <a href="http://doi.org/10.1093/gbe/evx240">Levchenko et al. 2018</a> noted that of the 3500 candidates identified so far, most are in non-coding regions, estimates of the fraction under positive selection range from 15%-85%, many are active in the brain, consistent with this major phenotypic difference between humans and chimpanzees, and about 7-8% are not shared with Neanderthals or Denisovans, consistent with the time of their divergence from the modern human lineage.</p>
</div>
<div id="recent-human-evolution" class="section level3">
<h3>Recent human evolution</h3>
<p>Human <span class="math inline">\(N_e\)</span> started a dramatic increase some 40-50,000 years ago as humans entered multiple new environments. <a href="https://doi.org/10.1073/pnas.0707650104">Hawks et al. (2007)</a> predicted and found that “selection has accelerated greatly during the last 40,000 years.” Co-authors Cochran and Harpending followed up with a book, <em>The 10,000 year explosion: How civilization accelerated human evolution</em> (you can find <a href="http://anthro.vancouver.wsu.edu/media/PDF/Hagen_2009_Human_natures_--_A_review_of_the_10000_year_explosion.pdf">my review of it here</a>). <a href="https://doi.org/10.1126/science.aaf5098">Fan et al. (2016)</a> review many examples of recent, population-specific human adaptations to local environments, such as the arctic, tropical rainforests, and high altitude:</p>
<div class="figure"><span id="fig:recentlocal"></span>
<img src="/images/Fan2016.png" alt="Examples of human local adaptations, each labeled by the phenotype and/or selection pressure, and the genetic loci under selection. Figure and caption from [Fan et al. (2016)](https://doi.org/10.1126/science.aaf5098)."  />
<p class="caption">
Figure 13: Examples of human local adaptations, each labeled by the phenotype and/or selection pressure, and the genetic loci under selection. Figure and caption from <a href="https://doi.org/10.1126/science.aaf5098">Fan et al. (2016)</a>.
</p>
</div>
</div>
</div>
<div id="wrap-up" class="section level2">
<h2>Wrap up</h2>
<p>Evolutionary anthropologists and human behavioral ecologists investigate the relationships between human environments and humans’ intricate molecular, cellular, anatomical, and behavioral phenotypes, most of which we inherited from primate, mammalian and earlier ancestors that evolved long before our lineage experienced a low <span class="math inline">\(N_e\)</span>. The extent to which a small <span class="math inline">\(N_e\)</span> limited human-specific evolution is still under investigation; it certainly didn’t reduce it to zero. And there are many other factors that might have accelerated it. There is abundant phenotypic evidence for human-specific adaptations, after all, and the genetic evidence so far indicates that positive selection played an important role in our evolutionary history.</p>
<p>Reading through the papers I discussed here, I was encouraged that folks studying human genetic variation seemed interested in, not hostile to, the many adaptationist hypotheses put forward by evolutionary anthropologists and behavioral ecologists. <a href="http://dx.doi.org/10.1038/hdy.2016.55">Charlesworth and Charlesworth</a>, for instance, highlight the importance of kin selection and evolutionary game theory, two theoretical foundations of behavioral ecology. <a href="https://dx.doi.org/10.1038%2Fnrg3336">O’Bleness et al. (2012)</a> give a nod to Lieberman’s endurance running hypothesizes and many other phenotypic comparisons of humans and non-human primates.</p>
<p>So what’s up with the SDSU biologists? The debates over adaptationism and neutralism vs. selectionism have all the hallmarks of a sectarian conflict. Contrary to our rhetoric, academics are among the most ethnocentric of an infamously ethnocentric species. Perhaps — dare I say it? — ethnocentrism is part of <a href="/pdf/ANTHProposal.pdf">human nature</a>. If so, it’s true of us all.</p>
<p>But, among the primates, our species also evolved a unique ability for <a href="https://doi.org/10.1086/203952">group alliances</a>. The SDSU biologists are doing <a href="http://www.bio.sdsu.edu/faculty/faculty.html">cool stuff</a>. Casey is doing <a href="https://scholar.google.com/citations?user=Oln60WcAAAAJ&amp;hl=en&amp;oi=ao">cool stuff</a>. Knowing Casey as I do, he doesn’t need my advice, but I’ll give it anyway: I doubt the SDSU biologists speak with one voice. There is as much diversity of opinion within groups as there is between them, maybe more. You are one of the very few SDSU social scientists who has a professional interest in what the SDSU biologists are doing. I suspect many of them recognize that.</p>
<!-- >The only nonmonogamous, group-living primates in which individuals of the larger sex are able to mount effective collective territorial action are either cooperative breeders or, if living in large fission–fusion groups, male bonded species. In any species where these 2 features co-occur, we should therefore expect pronounced collective territoriality. Indeed, among primates, this is uniquely the case in humans....([Willems and van Schaik 2015](https://doi.org/10.1093/beheco/arv001)). -->
<hr />
<!-- For a review of experimental evolution studies, see [Bailey and Bataillon (2016)](http://doi.org/10.1111/mec.13378). -->
<!-- Table 1: A compilation of 4332 genes that have reported to contain signatures of selection. Hominid Lineage: Summarizes genes identified by comparative methods (i.e., divergence) between human and other primates or mammals. Only genes that show lineage-specific selection in humans are listed. Early Modern Human: Summarizes genes identified by comparing sequences between modern human and Neanderthal. Human Population: Summarizes genes identified by genome-wide scans of recent positive selection in humans using polymorphism data. References are provided for each listed study at the end of the table. -->
</div>

    </div>
  </article>

  
<section id="comments">
  <div id="disqus_thread"></div>
  <script>
  var disqus_config = function () {
  
  };
  (function() {
    var inIFrame = function() {
      var iframe = true;
      try { iframe = window.self !== window.top; } catch (e) {}
      return iframe;
    };
    if (inIFrame()) return;
    var d = document, s = d.createElement('script');
    s.src = '//grasshoppermouse.disqus.com/embed.js'; s.async = true;
    s.setAttribute('data-timestamp', +new Date());
    (d.head || d.body).appendChild(s);
  })();
  </script>
  <noscript>Please enable JavaScript to view the <a href="https://disqus.com/?ref_noscript">comments powered by Disqus.</a></noscript>
</section>



</main>

      <footer class="footer">
        <ul class="footer-links">
          <li>
            <a href="/index.xml" type="application/rss+xml" target="_blank">RSS feed</a>
          </li>
          <li>
            <a href="https://gohugo.io/" class="footer-links-kudos">Made with <img src="/images/hugo-logo.png" width="22" height="22"></a>
          </li>
        </ul>
      </footer>

    </div>
    

    
<script src="/js/math-code.js"></script>
<script async src="//cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-MML-AM_CHTML"></script>


    
<script>
(function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
})(window,document,'script','https://www.google-analytics.com/analytics.js','ga');

ga('create', 'UA-59181618-1', 'auto');
ga('send', 'pageview');
</script>

  </body>
</html>

