<!DOCTYPE html>
<html lang="en-us">
  <head>
    <meta charset="utf-8">
<meta name="viewport" content="width=device-width, initial-scale=1.0">
<meta name="generator" content="Hugo 0.30" />


<title>About 90% of the genome is junk, which is very informative about ancestry but says little about biology - Ed Hagen</title>
<meta property="og:title" content="About 90% of the genome is junk, which is very informative about ancestry but says little about biology - Ed Hagen">



  







<link rel="stylesheet" href="/css/fonts.css" media="all">
<link rel="stylesheet" href="/css/main.css" media="all">



  </head>
  <body>
    <div class="wrapper">
      <header class="header">
        <nav class="nav">
  <a href="/" class="nav-logo">
    <img src="/images/logo.png"
         width="200"
         height="300"
         alt="Logo">
  </a>

  <ul class="nav-links">
    
    <li><a href="/about/">About</a></li>
    
    <li><a href="http://anthro.vancouver.wsu.edu/faculty/hagen/">Ed Hagen@WSU</a></li>
    
  </ul>
</nav>

      </header>


<main class="content" role="main">

  <article class="article">
    
    <span class="article-duration">15 min read</span>
    

    <h1 class="article-title">About 90% of the genome is junk, which is very informative about ancestry but says little about biology</h1>

    
    <span class="article-date">2019/07/27</span>
    

    <div class="article-content">
      <style>
p.caption {
  font-size: 0.7em;
}
</style>
<p>The genome, taken as a whole, has a profound influence on our biology. Many scientists, myself included, see it as a blueprint for the organism, or perhaps more accurately, as <a href="/2018/06/25/the-universal-genetic-program-and-the-custom-built-phenotype-implications-for-race-and-sex/">I explain here</a>, a program that generates the entire organism. Why, then, have many geneticists concluded that about 90% of the human genome has no influence on the human organism whatsoever, that it’s “junk”? And if most of the genome is junk, how can it reveal anything about an individual’s ancestry?</p>
<p>I’ll explain why most genetic differences among individuals and populations are probably differences in junk DNA, and therefore, perhaps counter-intuitively, are incredibly informative about ancestry but say little about biology.</p>
<div id="why-is-most-of-the-genome-thought-to-be-junk" class="section level1">
<h1>Why is most of the genome thought to be junk?</h1>
<div id="clue-1-huge-variation-in-eurkaryotic-genome-size" class="section level2">
<h2>Clue #1: huge variation in eurkaryotic genome size</h2>
<p>In the 1940’s and 50’s, biologists determined that the quantity of DNA was remarkably <em>constant</em> in cells from the same species, and therefore dubbed this quantity the “C-value”, but that it was quite variable across cells from different species. Importantly, the C-value, which we now know is the size of the genome (i.e., the number of nucleotides, or “letters”), bears little obvious relationship to the complexity of the organism. Some amoeba, for instance, which are single-celled organisms, have genomes that are 100x larger than the human genome (<a href="https://www.cell.com/current-biology/pdf/S0960-9822(12)01154-2.pdf">Eddy 2012</a>). The human genome contains eight times more DNA than that of a puffer fish but 40 times less than that of a lungfish, and among the &gt;200 salamander genomes analyzed as of 2014, all are between 4 and 35 times larger than the human genome (<a href="https://doi.org/10.1371/journal.pgen.1004351">Palazzo and Gregory 2014</a>). See Figure <a href="#fig:cvalue">1</a> (note the logarithmic scale).</p>
<div class="figure"><span id="fig:cvalue"></span>
<img src="/images/journal.pgen.1004351.g001.png" alt="Summary of haploid nuclear DNA contents (“genome sizes”) [in megabases] for various groups of eukaryotes. This graph is based on data for about 10,000 species. There is a wide range in genome sizes even among developmentally similar species, and there is no correspondence between genome size and general organism complexity. Humans, which have an average-sized genome for a mammal, are indicated by a star. **Note the logarithmic scale.** Figure and caption from [Palazzo and Gregory 2014](https://doi.org/10.1371/journal.pgen.1004351)."  />
<p class="caption">
Figure 1: Summary of haploid nuclear DNA contents (“genome sizes”) [in megabases] for various groups of eukaryotes. This graph is based on data for about 10,000 species. There is a wide range in genome sizes even among developmentally similar species, and there is no correspondence between genome size and general organism complexity. Humans, which have an average-sized genome for a mammal, are indicated by a star. <strong>Note the logarithmic scale.</strong> Figure and caption from <a href="https://doi.org/10.1371/journal.pgen.1004351">Palazzo and Gregory 2014</a>.
</p>
</div>
<p>Why would the “blueprints” of some simple organisms be so much larger than those of more complex ones, and why would the genomes of very closely related species often differ so dramatically? It is hard to escape the conclusion that a substantial fraction of a large genome has little, if any, influence on the phenotype of the organism. What, then, explains the expansion in genome size in some lineages?</p>
</div>
<div id="clue-2-pervasive-parasitic-dna" class="section level2">
<h2>Clue #2: pervasive parasitic DNA</h2>
<p>A key property of DNA is that it can copy itself. Typically, this serves to make a <em>new</em> copy of the genome in a daughter cell, but sometimes sections of DNA copy themselves from one part of the genome into another, a type of mutation that often causes disease (<a href="https://doi.org/10.1186/s13100-016-0065-9">Hancks &amp; Kazazian, 2016</a>). These sequences of DNA, termed <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2874221/"><em>transposons</em></a>, are considered to be <a href="https://en.wikipedia.org/wiki/Selfish_genetic_element"><em>selfish</em> or <em>parasitic</em> DNA</a> because they have evolved features that increase their own replication at the expense of other genes in the genome, e.g., by inserting themselves into or near a gene and thus disrupting its function, consequently harming the organism itself.</p>
<p>There are two major classes of transposons. <a href="https://en.wikipedia.org/wiki/Transposable_element#DNA_transposons">DNA transposons</a> operate by a <em>cut</em>-and-paste mechanism – so-called “jumping genes” – and make up only about 3% percent of the human genome. <a href="https://en.wikipedia.org/wiki/Retrotransposon">Retrotransposons</a>, in contrast, operate by a <em>copy</em>-and-paste mechanism, and therefore increase the size of the genome (see Figure <a href="#fig:retrotransposon">2</a>). Retrotransposons make up at least 45% of the human genome (<a href="https://doi.org/10.1038/nrg2640">Cordaux and Batzer 2009</a>).</p>
<div class="figure"><span id="fig:retrotransposon"></span>
<img src="/images/retrotransposons.png" alt="Retrotransposon. Figure from [Wikimedia.org](https://commons.wikimedia.org/wiki/File:Retrotransposons.png)"  />
<p class="caption">
Figure 2: Retrotransposon. Figure from <a href="https://commons.wikimedia.org/wiki/File:Retrotransposons.png">Wikimedia.org</a>
</p>
</div>
<p><a href="https://en.wikipedia.org/wiki/Alu_element"><em>Alu</em></a> and <a href="https://en.wikipedia.org/wiki/LINE1">LINE-1</a> retrotransposons in the human genome insert perhaps as frequently as once every 20 births (<a href="https://doi.org/10.1038/nrg2640">Cordaux and Batzer 2009</a>). Like other parasites, these copies can themselves make copies, and thus pose an extreme threat to other genes and to the organism itself. Many mechanisms have therefore evolved to neutralize them <a href="https://doi.org/10.1186/s13100-016-0070-z">(Goodier, 2016)</a>. To illustrate: due to their activity over the last 150 million years, there are &gt;500,000 <a href="https://en.wikipedia.org/wiki/LINE1">LINE-1</a> elements in the human genome, but less than 100 copies are currently active, i.e., able to replicate within the genome (<a href="https://doi.org/10.1038/nrg2640">Cordaux and Batzer 2009</a>).</p>
<p>The emerging picture, then, is that over evolution, lineages of retrotransposons have expanded, enlarging the genome (see Figure <a href="#fig:aluexpansion">3</a>). Inserted elements that disrupted gene function were removed by <a href="https://en.wikipedia.org/wiki/Negative_selection_(natural_selection)">negative selection</a> (<a href="https://doi.org/10.1080/2159256X.2017.1280116">Rishishwar et al., 2017</a>). Others were suppressed by cellular mechanisms, and then began to degrade due to accumulating mutations, becoming what we now call junk DNA (<a href="https://doi.org/10.1080/2159256X.2017.1280116">Rishishwar et al., 2017</a>), although there is solid evidence that transposable elements have often been co-opted for the regulation of host genes (<a href="https://doi.org/10.1038/nrg.2016.139">Chuong et al. 2017</a>). Some particularly ancient retrotransposons might have degraded so extensively that they are no longer easily recognized. One analysis estimated that perhaps up to 70% of the human genome comprises such repetitive elements <a href="https://doi.org/10.1371/journal.pgen.1002384">(Koning, Gu, Castoe, Batzer, &amp; Pollock, 2011)</a>.</p>
<div class="figure"><span id="fig:aluexpansion"></span>
<img src="/images/aluexpansion.png" alt="The expansion of Alu elements in primates. The expansion of Alu subfamilies (Yc1, Ya5a2, Yb9, Yb8, Y, Sg1, Sx and J) is superimposed on a tree of primate evolution. The expansion of the various Alu subfamilies is colour coded to denote the times of peak amplification. The approximate copy numbers of each Alu subfamily are also noted. Mya, million years ago. Figure and caption from [Batzer and Deininger 2002](https://doi.org/10.1038/nrg798)."  />
<p class="caption">
Figure 3: The expansion of Alu elements in primates. The expansion of Alu subfamilies (Yc1, Ya5a2, Yb9, Yb8, Y, Sg1, Sx and J) is superimposed on a tree of primate evolution. The expansion of the various Alu subfamilies is colour coded to denote the times of peak amplification. The approximate copy numbers of each Alu subfamily are also noted. Mya, million years ago. Figure and caption from <a href="https://doi.org/10.1038/nrg798">Batzer and Deininger 2002</a>.
</p>
</div>
</div>
<div id="clue-3-which-was-misleading-only-1-2-of-dna-codes-for-protein" class="section level2">
<h2>Clue #3 (which was misleading): only 1-2% of DNA codes for protein</h2>
<p>The genetic code was cracked in the 1960’s: triplets of nucleotides (A,C,G,T) code for specific amino acids, the building blocks of proteins. Only about 1-2% of the genome codes for proteins, however. It was therefore often reported that the remaining 98-99% of the genome was junk, even though it was known <a href="https://en.wikipedia.org/wiki/Lac_operon">as early as 1961</a> that some non-coding DNA regulated the expression of protein-coding DNA. By the early 2000’s, however, it was widely recognized that much non-coding DNA plays a critical regulatory role <a href="https://doi.org/10.1101/gr.6339607">(Gerstein et al., 2007)</a>.</p>
<p>Thus, more than 1-2% of the genome is functional, but how much more?</p>
</div>
<div id="clue-4-about-90-of-the-genome-shows-no-evidence-of-sequence-conservation" class="section level2">
<h2>Clue #4: about 90% of the genome shows no evidence of sequence conservation</h2>
<p>The human genome has about 3 billion nucleotides. Every time the genome is copied, each nucleotide has a small probability that it will be miscopied, i.e., will mutate. In humans, there are up to 100 such mutations with each birth. Mutations that alter organism functions will undergo positive or negative selection. Because most such mutations are harmful — there are many more ways to break something than to improve it — they will typically disappear from the population via negative selection. A fertilized zygote with a mutation in a critical part of the genome might not even successfully develop. DNA sequences in which most mutations face negative selection are termed <em>conserved</em> or <em>constrained</em>.</p>
<p>Mutations that do not alter organism function, on the other hand, will not experience positive or negative selection and will therefore accumulate in junk DNA at a more-or-less constant rate. Functional DNA can therefore be distinguished from junk DNA by its greater degree of conservation across species.</p>
<p>Protein coding sequences exhibit a surprising degree of conservation. Yeast and humans diverged about a billion years ago, for example, yet 23% of yeast genes have homologs in humans (i.e., the genes derive from a common ancestor). Of these genes, the amino acid sequences overlap, on average, by about 32%. Even more remarkable, after replacing 414 critical yeast genes with their human homologs, 47% of the human genes functioned and enabled the yeast to survive <a href="https://doi.org/10.1126/science.aaa0769">(Kachroo et al. 2015)</a>.</p>
<p>A disadvantage of methods that assess DNA functionality by comparing degrees of sequence conservation across species is that they cannot identify functional sequences that might be recently acquired in a particular species. There might be functional sequences of DNA in the human genome, for instance, that do not appear in the genomes of chimps or other primates.</p>
<p>To assess what fraction of the human genome has been subject to negative selection on both long and short timescales, i.e., is constrained and is therefore likely to be functional, <a href="http://dx.doi.org/10.1371/journal.pgen.1004525">Rand et al. (2014)</a> analyzed DNA sequences across the mammals in several functional classes (i.e., protein coding sequences vs. various types of regulatory sequences that might have been acquired recently). They found that 7.1– 9.2% of the human genome is presently constrained, fairly consistent with previous results that found that between 3% and 15% of the human genome was functional. As expected, protein coding sequences were highly constrained, whereas regulatory sequences experienced more rapid turnover (see Figure <a href="#fig:conserved">4</a>). This implies, of course, that &gt;90% of human DNA is not functional, i.e., is junk (for similar recent estimates, see <a href="https://doi.org/10.1126/science.1225057">Ward and Kellis, 2012</a>; <a href="https://doi.org/10.1038/ng.3196">Gulko et al., 2015</a>; and references therein).</p>
<div class="figure"><span id="fig:conserved"></span>
<img src="/images/journal.pgen.1004525.g004.png" alt="Model-based inference of turnover by functional class [across the mammals]. Schematic summary of the fraction of constrained sequence that has been retained (saturated colours) or turned over (pastel colours) in the human lineage over time (X-axis, divergence time) and how it has been distributed across various categories of functional element. In addition to showing the reduced quantity of preserved constrained sequence with increasing divergence, we infer the reciprocal quantity of sequence that is assumed to have been gained over human lineage evolution. For consistency this approach requires mutually exclusive annotation sets, in contrast to those used in Figure 3, making the results not directly comparable. Overlaps between the major different annotations are shown in Figure S10. Figure and caption from [Rands et al. 2014](http://dx.doi.org/10.1371/journal.pgen.1004525)."  />
<p class="caption">
Figure 4: Model-based inference of turnover by functional class [across the mammals]. Schematic summary of the fraction of constrained sequence that has been retained (saturated colours) or turned over (pastel colours) in the human lineage over time (X-axis, divergence time) and how it has been distributed across various categories of functional element. In addition to showing the reduced quantity of preserved constrained sequence with increasing divergence, we infer the reciprocal quantity of sequence that is assumed to have been gained over human lineage evolution. For consistency this approach requires mutually exclusive annotation sets, in contrast to those used in Figure 3, making the results not directly comparable. Overlaps between the major different annotations are shown in Figure S10. Figure and caption from <a href="http://dx.doi.org/10.1371/journal.pgen.1004525">Rands et al. 2014</a>.
</p>
</div>
</div>
</div>
<div id="the-encode-controversy" class="section level1">
<h1>The ENCODE controversy</h1>
<p><a href="https://www.encodeproject.org">ENCODE</a> is a large consortium of research groups whose <a href="https://www.encodeproject.org/help/project-overview/">goal</a> is to “build a comprehensive parts list of functional elements in the human genome, including elements that act at the protein and RNA levels, and regulatory elements that control cells and circumstances in which a gene is active.” Their methods for identifying function primarily involve detecting evidence that a nucleotide participates in a biochemical RNA- and/or chromatin-associated event.</p>
<p>In 2012, ENCODE <a href="https://doi.org/10.1038/nature11247">published</a> their first analysis of functionality across the entire human genome. This was an enormously important milestone, but one claim stood out:</p>
<blockquote>
<p>“These data enabled us to assign biochemical functions for 80% of the genome” (p. 57).</p>
</blockquote>
<p>ENCODE appeared to be challenging all the evidence I’ve reviewed here that most of the genome is junk. Several news commentaries trumpeted the death of the junk DNA concept (see refs in <a href="https://doi.org/10.1093/gbe/evt028">Graur et al., 2013</a>).</p>
<p>Biting critiques soon followed (<a href="https://doi.org/10.1016/j.cub.2012.10.002">Eddy, 2012</a>; <a href="https://doi.org/10.1073/pnas.1221376110">Doolittle, 2013</a>; <a href="https://doi.org/10.1093/gbe/evt028">Graur et al., 2013</a>; <a href="10.1016/j.bbrc.2012.12.074">Niu &amp; Jiang, 2013</a>; and <a href="https://doi.org/10.1371/journal.pgen.1004351">Palazzo &amp; Gregory, 2014</a>). These researchers noted that ENCODE employed an unusually broad definition of “function” based on the biochemical effects of nucleotide sequences, regardless of whether those effects evolved by natural selection. By ENCODE standards, the thumping sound of the heart is a “function” of the heart because it has detectable biological effects, and the disabling effects of dangerous genetic birth defects are also “functional” (e.g., <a href="https://doi.org/10.1093/gbe/evt028">Graur et al., 2013</a>). ENCODE’s approach does have some defenders, however (e.g., <a href="https://doi.org/10.1007/s10539-014-9441-3">Germain et al., 2014</a>).</p>
<p>Much of the ENCODE claim of functionality rests on evidence that a sequence is <a href="https://en.wikipedia.org/wiki/Transcription_(biology)">transcribed</a> into RNA. According to the critics, transcription is an inherently noisy process that occurs in most of the genome regardless of evolved functionality, and that most transcripts are probably junk (e.g., <a href="https://doi.org/10.3389/fgene.2015.00002">Palazzo and Lee, 2015</a>), although here, too, there is push back (e.g., <a href="https://doi.org/10.1016/j.tig.2017.08.002">Jandura and Krause, 2017</a>).</p>
<p>Although the ENCODE claim of 80% functionality has found some defenders, I favor a definition of “function” that is based on natural selection, and therefore remain skeptical that most of the genome is functional.</p>
</div>
<div id="why-is-junk-dna-ideal-for-revealing-ancestry" class="section level1">
<h1>Why is junk DNA ideal for revealing ancestry?</h1>
<p>If most of the human genome is junk DNA, as it appears to be, then most genetic differences among individuals and populations are differences in junk DNA that have no biological significance. It turns out, though, that lack of biological significance is ideal for inferring ancestry.</p>
<p>Every person inherits half their DNA from each parent. Each parent, in turn, inherited half their DNA from each of their two parents, and so forth. Thus, each section of one’s DNA has its own unique lineage of ancestors. Due to random copying errors, a small number of changes in DNA sequence are introduced with each transmission from parent to offspring. Over generations each section of DNA will therefore acquire a unique “fingerprint” of variations. Sections of DNA that are inherited from a recent common ancestor will have very similar fingerprints, whereas those inherited from a more distant common ancestor will have more distinct fingerprints. By comparing an individual’s sequences with a large database of sequences across the entire genome, from individuals living around the world, it is possible to estimate which sequences of DNA share recent common ancestors from which locations. Because random errors accumulate with each generation, the degree of similarity or difference also provides an estimate of the number of generations, and hence years, to those common ancestors.</p>
<p>To illustrate with my two daughters: They each inherited one chromosome 6 from me, and one from my wife. My dad has Norwegian ancestry and my mom has Ashkenazi Jewish ancestry. So, with 50-50 probability (and ignoring <a href="https://en.wikipedia.org/wiki/Genetic_recombination">recombination</a>), the chromosome 6 they inherited from me might more closely resemble chromosome 6’s from Northern Europe or those from central European Jewish communities. My wife has Mexican American background, with a great-great grandparent with African background. So the chromosome 6 they inherited from her might more closely resemble those of Spanish, Portuguese, or other Europeans (with 90.6% probability), Native Americans (with 6.25% probability), or Africans (with 3.125% probability). See Figure <a href="#fig:globalancestry">5</a>.</p>
<div class="figure"><span id="fig:globalancestry"></span>
<img src="/images/globalancestry.png" alt="Global Ancestry. The green arrows symbolize migration of early human ancestors out of Africa. The color mosaic denotes global population diversity resulting from various subsequent inter- and intra-continental and regional migrations. The pedigree represents the complex network of intermediate and recent ancestors that is the subject of individual genetic genealogy testing. Figure and caption from [Royal et al. 2010](https://doi.org/10.1016/j.ajhg.2010.03.011)."  />
<p class="caption">
Figure 5: Global Ancestry. The green arrows symbolize migration of early human ancestors out of Africa. The color mosaic denotes global population diversity resulting from various subsequent inter- and intra-continental and regional migrations. The pedigree represents the complex network of intermediate and recent ancestors that is the subject of individual genetic genealogy testing. Figure and caption from <a href="https://doi.org/10.1016/j.ajhg.2010.03.011">Royal et al. 2010</a>.
</p>
</div>
<p>Figure <a href="#fig:ancestry">6</a> is the output of one popular computer program for determining ancestry from DNA sequences. Each thin vertical line is one individual, and the different colors represent the fractions of their DNA sequences that more closely match individuals from different regions. My daughters would be mostly green, with some purple and a dash of red. President Obama would (probably) have a line that is about half green and half red.</p>
<div class="figure"><span id="fig:ancestry"></span>
<img src="/images/ancestry.jpg" alt="Regional ancestry inferred with the frappe program at K = 7 (13) and plotted with the Distruct program (31). Each individual is represented by a vertical line partitioned into colored segments whose lengths correspond to his/her ancestry coefficients in up to seven inferred ancestral groups. Population labels were added only after each individual's ancestry had been estimated; they were used to order the samples in plotting. Figure and caption from [Li et al. 2009](https://doi.org/10.1126/science.1153717)."  />
<p class="caption">
Figure 6: Regional ancestry inferred with the frappe program at K = 7 (13) and plotted with the Distruct program (31). Each individual is represented by a vertical line partitioned into colored segments whose lengths correspond to his/her ancestry coefficients in up to seven inferred ancestral groups. Population labels were added only after each individual’s ancestry had been estimated; they were used to order the samples in plotting. Figure and caption from <a href="https://doi.org/10.1126/science.1153717">Li et al. 2009</a>.
</p>
</div>
<p>Such inferences of ancestry for each section of DNA assume that the patterns of variation in those sections result only from descent with <em>random</em> copying errors. These assumptions hold for junk DNA. If there were processes that caused sequences with distant common ancestors to closely resemble each other, however, or sequences with recent common ancestors to sharply diverge from one another, a geneticist might mistakenly conclude that some sequences have more recent vs. more distant common ancestors than they actually do, and incorrectly infer ancestry. In functional DNA, this can occur due to positive, negative, or frequency dependent selection, i.e., factors related to the environment rather than ancestry.</p>
<p>To give one example, distantly related pastoralist populations in Africa and Europe independently evolved very similar sequence variants in the promoter region of the lactase gene on chromosome 2 that allow adults to digest lactose, a remarkable example of recent convergent evolution under very strong positive selection in humans (<a href="https://doi.org/10.1038/ng1946">Tishkoff et al. 2007</a>). A naive analysis might mistakenly infer that some distantly related European and African individuals had a recent common ancestor for this section of chromosome 2.</p>
<p>In short, in functional DNA, patterns of variation due to ancestry (and also to population history) are confounded with patterns of variation due to natural selection (e.g., <a href="https://doi.org/10.1371/journal.pbio.0020286">Akey et al. 2004</a>, <a href="https://doi.org/10.1126/science.1124309">Sabeti et al. 2006</a>, <a href="https://doi.org/10.1101/gr.088336.108">Nielsen et al. 2009</a>).</p>
<p>Using pure junk DNA to infer ancestry would be ideal, and several studies have used <em>Alu</em> insertions to investigate human ancestry and demography (<a href="https://doi.org/10.1080/2159256X.2017.1280116">Rishishwar et al., 2017</a>). Functional and non-functional DNA are intimately intertwined in the genome, however, and it is not entirely clear which non-coding sequences are functional and which are not. Ancestry is therefore typically inferred using common genetic variants across the genome (for a readable introduction to inferring ancestry from genetic data, see <a href="https://doi.org/10.1016/j.ajhg.2010.03.011">Royal et al., 2010</a>). Given that most DNA is junk, most common genetic variants should be junk, but are they?</p>
<p>Many studies have attempted to determine the functional consequences, if any, of coding and noncoding variants. Because the human genome is so <a href="/2018/06/25/the-universal-genetic-program-and-the-custom-built-phenotype-implications-for-race-and-sex/">poorly understood</a>, these studies typically combine diverse sources of sequence, evolutionary, and functional information using use neural network and other machine learning methods to classify the variants according to their functional effects. One effort, <a href="https://doi.org/10.1073/pnas.1216613110">Bromberg et al. (2013, p. 14255)</a>, concluded that:</p>
<blockquote>
<p>[V]ariants in seemingly healthy individuals tend to be neutral or weakly disruptive for protein molecular function. These variant effects are predicted to be largely either experimentally undetectable or are not deemed significant enough to be published. This may suggest that nondisease phenotypes arise through combinations of many variants whose effects are weakly nonneutral (damaging or enhancing) to the molecular protein function but fall within the wild-type range of overall physiological function.</p>
</blockquote>
<p>A recent review of methods assessing phenotypic effects of noncoding variants, on the other hand, was skeptical of their performance, concluding “A significant disconnect is found to exist between the statistical modelling and biological performance of predictive approaches” (<a href="https://doi.org/10.1038/s41467-018-08270-y">Liu et al. 2019</a>). Part of the problem is that the methods stumble with the severe class imbalance: the small number of variants with meaningful effects among a relatively large number of variants without such effects.</p>
<p>In summary, evidence to date is consistent with the view that most human DNA sequences are junk and are evolving neutrally. Variants in these sequences are yielding a rich trove of information on human ancestry and population history, but don’t tell us much about our biology.</p>
<p>None of the foregoing calls into question the profound importance of <em>functional</em> DNA to human biology. I write about <a href="/2018/06/25/the-universal-genetic-program-and-the-custom-built-phenotype-implications-for-race-and-sex/">that here</a>, highlighting a perspective on the genome that hasn’t gotten much press. And <a href="/2018/03/03/while-it-may-be-true-that-evolutionary-anthropologists-consider-themselves-scientists/">here I write about</a> the relative influences of positive natural selection vs. random genetic drift on the evolution of the human lineage.</p>
</div>
<div id="concluding-remarks" class="section level1">
<h1>Concluding remarks</h1>
<p>Each of our many ancestors indisputably lived at certain times and in certain places, and it’s pretty amazing that we can now glean those hard facts from spit in a tube. Living in a certain time and place, however, in and of itself, says nothing about the biology of those ancestors.</p>
<!-- Two typical humans differ from one another at about 5 million locations across their genomes ([The 1000 Genomes Project Consortium, 2015](https://doi.org/10.1038/nature15393)). By comparing one individual's patterns of genetic differences to large databases of such differences in millions of other individuals whose ancestries are known, it is possible to make increasingly accurate estimates of the geographic locations of that individual's ancestors.  -->
<!-- What do these genetic differences, which are so informative about ancestry, reveal about differences in biology? Not much, it turns out. Roughly 90% of the genome has no influence on the human phenotype, i.e., no influence on our biology. Therefore, most of the genetic differences between individuals say nothing about biology. -->
<!-- This blog post will explain why most geneticists have concluded that a large fraction of the genome is "junk", i.e., has no influence on our phenotype. First, though, I want to explain, via a simple analogy, how junk DNA can nevertheless be extraordinarily informative about ancestry.  -->
<!-- To sketch how this works, imagine that the [flying spaghetti monster](https://en.wikipedia.org/wiki/Flying_Spaghetti_Monster) assigned Adam and Eve unique personal identification codes comprising random letters. These codes had no more relation to the phenotypes or biology of Adam and Eve than the digits of your social security or driver's license number relate to your biology. They were just random codes that were inscribed in their DNA. When Adam and Eve procreated, producing several children, each child inherited half of each parent's code, except that, randomly, a few letters were miscopied in each code when transmitted from parent to child. These children then marry and have their own children, and the same things happen. -->
<!-- A geneticist then sequences the DNA of the grandchildren. She groups the grandchildren with the most similar codes together, inferring that they had the same parents. She then compares these groups to determine which have the most similar codes, inferring that they had the same parents, in this case Adam and Eve, and thus that the grandchildren have "Garden of Eden" ancestry. By estimating how many random changes have occurred in descendants'' codes relative to their ancestors', the geneticist can also estimate the number of generations, and therefore years, that separate them. These inferences work because the geneticist can assume that the grandchildren acquire half their codes from each parent, with a few random changes in each half due to copying errors, i.e., are only influenced by factors related to ancestry. These assumptions hold for junk DNA.  -->
<!-- To illustrate: the [FOXP2 gene](https://en.wikipedia.org/wiki/FOXP2) plays some critical role in development (perhaps having to do with language). There is very strong negative selection on this gene, and mutations are extraordinarily rare in the population. Based on FOXP2 sequence alone, distantly related individuals would erroneously seem to be closely related. -->
<!-- Or consider the sickle hemoglobin (HbS) allele that is common in regions of Africa and South Asia with endemic malaria, to which it provides protection in heterozygotes. But because homozygotes for this allele suffer a serious disease, it is also subject to frequency dependent selection (for review, see [Piel et al. 2010](https://doi.org/10.1038/ncomms1104)). Patterns of variation in Hb and HbS therefore reflect a complex combination of ancestry, environmental factors (presence or absence of malaria), and frequency-dependent selection. HbS is one of few variants in functional DNA for which we understand these factors. The factors affecting most functional variants are very poorly understood. -->
<!-- In practice there are many complications and pitfalls. For one thing, functional and nonfunctional regions of the genome are intimately intertwined. Nevertheless, determining ancestry from DNA sequences is basically the same as the Adam and Eve example: computer programs compare DNA sequences from a particular individual --- ideally only junk DNA sequences --- to databases of such sequences from folks around the globe, and identify which sequences mostly closely resemble those of people living here vs. there. Half of Obama's sequences of letters most closely resemble sequences of letters of folks living in Africa, for example, whereas the other half most closely resemble sequences of letters of folks living in Europe. -->
<!-- That's about all there is to it. As long as changes in DNA sequences only reflect mating and descent with a few random copying errors, and as long as there is a sufficiently detailed database of individual sequences that are linked to geographic locations, it is possible to estimate the dates and locations of one's ancestors. As the size and diversity of the databases increase, the accuracy of these estimates increase.  -->
<!-- 
I've signed 100% of my reviews. Author of first paper I reviewed as a grad student, themselves a newly minted PhD, got pissed and complained to journal that they dare assign a grad student to review. Later, interviewed for a job where said author was faculty. They were still pissed. Didn't get the job. 
-->
</div>

    </div>
  </article>

  
<section id="comments">
  <div id="disqus_thread"></div>
  <script>
  var disqus_config = function () {
  
  };
  (function() {
    var inIFrame = function() {
      var iframe = true;
      try { iframe = window.self !== window.top; } catch (e) {}
      return iframe;
    };
    if (inIFrame()) return;
    var d = document, s = d.createElement('script');
    s.src = '//grasshoppermouse.disqus.com/embed.js'; s.async = true;
    s.setAttribute('data-timestamp', +new Date());
    (d.head || d.body).appendChild(s);
  })();
  </script>
  <noscript>Please enable JavaScript to view the <a href="https://disqus.com/?ref_noscript">comments powered by Disqus.</a></noscript>
</section>



</main>

      <footer class="footer">
        <ul class="footer-links">
          <li>
            <a href="/index.xml" type="application/rss+xml" target="_blank">RSS feed</a>
          </li>
          <li>
            <a href="https://gohugo.io/" class="footer-links-kudos">Made with <img src="/images/hugo-logo.png" width="22" height="22"></a>
          </li>
        </ul>
      </footer>

    </div>
    

    

    
<script>
(function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
})(window,document,'script','https://www.google-analytics.com/analytics.js','ga');

ga('create', 'UA-59181618-1', 'auto');
ga('send', 'pageview');
</script>

  </body>
</html>

